### PR TITLE
Fixed min depth corr calculation.

### PR DESCRIPTION
**Pull request by marcomuzio**
Accounted for the fact that the max correlation can be above the ice surface by removing the `z_thresh` condition. Fixed the calculation across maps to use the maximum correlation value across all maps as the reference value. Fixed the A1 calpulser radius.

### CODE REVIEW

File: araproc/analysis/standard_reco.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def __init__(self,
         # so look that up
 
         self.calpulser_r_library = {
-            100 : "49.48",
+            100 : "48.02",
```
Comment (clark2668) [inline araproc/analysis/standard_reco.py:99]: Is this intended? It keeps changing....

File: araproc/analysis/standard_reco.py [hunk 2]
```diff
@@ -803,16 +801,13 @@ def min_frac_corr_depth(self, corr_map, fraction=0.6, z_thresh=0):
                     depth = radius * math.sin(math.radians(theta)) + avg_z
                     
 
-                    # Ensure min_depth does not exceed z_thresh
-                    if depth > min_depth and depth <= z_thresh:
+                    # Update min_depth 
+                    if depth > min_depth:
```
Comment (clark2668) [inline araproc/analysis/standard_reco.py:805]: This goes the right way? In case depth is a negative number?

Comment (AlanSalcedo) [reply araproc/analysis/standard_reco.py:805]: Right. I remember this is sweeping downward from the surface (really from z_threshold). Maybe we need to sweep up for negative depths? 

I can do that today if you want

Comment (clark2668) [reply araproc/analysis/standard_reco.py:805]: I don't have strong opinions about which way you go! I'm really asking about the sign convention I guess.

File: araproc/analysis/standard_reco.py (hunk 1 above)
Comment (marcomuzio) [reply araproc/analysis/standard_reco.py:99]: Yes, this is restoring the value that was changed in [PR#42](https://github.com/ara-software/AraProc/pull/42). I did this based on the values Alex said to ignore/take seriously.

File: araproc/analysis/standard_reco.py (hunk 2 above)
Comment (marcomuzio) [reply araproc/analysis/standard_reco.py:805]: This should be right I think. A more negative depth is closer to Earth's center and you want the shallowest (i.e. most positive) depth where the conditions are satisfied.

**Comment by marcomuzio** [comment]
All happy with this?

File: araproc/analysis/standard_reco.py (hunk 2 above)
Comment (clark2668) [reply araproc/analysis/standard_reco.py:805]: Okay, then sounds good

File: araproc/analysis/standard_reco.py (hunk 1 above)
Comment (clark2668) [reply araproc/analysis/standard_reco.py:99]: Okay, great

**Comment by clark2668** [comment]
Merging sounds good to me!

**Comment by AlanSalcedo** [comment]
Looks good to me. Thanks Marco!

**Comment by marcomuzio** [comment]
Thanks guys!

**Pull request merged**